### ISSUE TITLE
tauri.conf.json seems not working when developing

### ISSUE THREAD

**Issue by jwenjian** [tauri-apps/tauri#705]
**Describe the bug**
When I run `npm run tauri:serve`, changing config values in `tauri.conf.json` seems no effect:

```json
"window": {
      "title": "Lucky You",
      "width": 800,
      "height": 700,
      "resizable": false,
      "fullscreen": false
    },
```

But, changing this file works when I run `npm run tauri:build`

**To Reproduce**
1. Create a vue project and add tauri plugin
2. Changing `window.*` items in `tauri.conf.json`
3. Run `npm run tauri:serve`
4. You will find that your change didn't make effect

**Expected behavior**
When I change the config in `tauri.conf.json`, especially the `window` object, it should take effect event when I'm in develop mode.

**Screenshots**
No

**Platform and Versions (please complete the following information):**

OS: MacOS 10.15.5
Node: 12.15.0
NPM: 6.13.4
Yarn:
Rustc: 1.44.1

**Additional context**
I pushed my code to [jwenjian/lucky-you](https://github.com/jwenjian/lucky-you), you can just clone it and run(if you have your dev env ready).

**Stack Trace**

**Comment by jasper-lyons** [comment]
Looks like the `src-tauri/target/debug` build isn't refreshed when the tauri.config.json changes.

When I `rm -r src-tauri/target/debug` and run `npx tauri dev` the changes to the `tauri.config.json` are picked up.

**Comment by lucasfernog** [comment]
A new Vue CLI Plugin was published this week, did you guys try it?

**Comment by jasper-lyons** [comment]
I'm not using Vue with my project, just raw html/css/js,

**Comment by jwenjian** [comment]
new Vue CLI plugin seems not working, `rm -f src-tauri/target/debug` works for me.

**Comment by lucasfernog** [comment]
I'm looking into it right now.

**Comment by lucasfernog** [comment]
Closed by #787

**Comment by jwenjian** [comment]
It works with tauri = 0.7.3 and tauri-api = 0.7.1 :tada:
